text/0000-better-env-handling.md

```diff
@@ -0,0 +1,89 @@
+- Start Date: 2020-04-23
+- Relevant Team(s): Ember.js, Learning
+- RFC PR: [#619](https://github.com/emberjs/rfcs/pull/619)
+- Tracking: (leave this empty)
+
+# Better environment handling
+
+## Summary
+
+This RFC proposes to add a tiny module similar to `@glimmer/env`, to allow to more easily work with common environment questions.
+
+## Motivation
```

Comment:
IMHO, this misses a very important issue / motivation:

Using `import { DEBUG } from '@glimmer/env'` is **inappropriate** for testing related guards because it means you _can not_ run your tests against production builds. I am 100% to blame for this issue (I added the `@glimmer/env` functionality, and kicked off a bunch of PRs to use `DEBUG` to guard testing related code) :sob:.

Reply:
Will add this as well!